addons/calendar/static/src/js/base_calendar.js

```diff
@@ -122,7 +122,18 @@ BasicModel.include({
      */
     _fetchSpecialAttendeeStatus: function (record, fieldName) {
```

Comment:
not sure this function has to be modified

Reply:
@Polymorphe57 I have modify this function for attendeeIDs, note that we are using this widget in calendar popover and calendar popover uses makeRecord method, in this case I will get attendeeIDs from ```ecord.data[fieldName].res_ids```